relstorage/storage.py

```diff
@@ -1073,6 +1073,9 @@ def _abort(self):
         if self.blobhelper is not None:
             self.blobhelper.abort()
 
+    def afterCompletion(self):
+        self._rollback_load_connection()
```

Comment:
It can be called unconditionally, no matter the state of the connection, and when its done you have a guarantee about the state (which is potentially helpful in the event of logic bugs). Callers that care about whether there may actually be something that needs rolled back check `_load_transaction_open` themselves, presumably for efficiency reasons.